api/server/server.go

```diff
@@ -130,6 +130,9 @@ func parseMultipartForm(r *http.Request) error {
 }
 
 func httpError(w http.ResponseWriter, err error) {
+	if err == nil || w == nil {
+		return
```

Comment:
yes, I'm also not sure whether we should keep this, but then I think it's better to keep this part, just in case, you know :P